spikeinterface/widgets/correlograms.py

```diff
@@ -44,16 +43,18 @@ def plot(self):
         for i, unit_id1 in enumerate(unit_ids):
             for j, unit_id2 in enumerate(unit_ids):
                 ccg = correlograms[i, j]
-                ax = self.axes[i, j]
+                ax = self.get_tiled_ax(i+j*len(unit_ids), len(unit_ids), len(unit_ids))
+                #ax = self.axes[i, j]
```

Comment:
This will break some code I think.

Reply:
Maybe but it had to be consistent with plot_waveforms and allow plots in ax axis